cmd/kube-score/object-checks-config-file.go

```diff
@@ -0,0 +1,112 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	flag "github.com/spf13/pflag"
+	"github.com/zegl/kube-score/config"
+	"github.com/zegl/kube-score/parser"
+	"github.com/zegl/kube-score/score"
+	"gopkg.in/yaml.v3"
+)
+
+type kubescorechecks struct {
+	AddAllDefaultChecks            bool     `yaml:"addAllDefaultChecks"`
+	AddAllOptionalChecks           bool     `yaml:"addAllOptionalChecks"`
+	DisableIgnoreChecksAnnotations bool     `yaml:"disableIgnoreChecksAnnotations"`
+	DefaultChecks                  []string `yaml:"defaultChecks"`
+	OptionalChecks                 []string `yaml:"optionalChecks"`
+	IncludeChecks                  []string `yaml:"include"`
+	ExcludeChecks                  []string `yaml:"exclude"`
+}
+
+func mkConfigFile(binName string, args []string) {
```

Comment:
mkConfigFile should return an error instead of panicing. Change the signature to `func mkConfigFile(binName string, args []string) error`, and create errors with `fmt.Errorf("....: %w", err)` if something fails.

And when calling `mkConfigFile` from main, check if it errored, and print the error and exit with code 1, similar to the other commands.